### ISSUE TITLE
CI - docker build (and publish) failing 

### ISSUE THREAD

**Issue by mccurdyc** [apideck-libraries/postman-to-k6#113]
Hello! 👋 

I noticed that you were using GitHub Container Registry from this PR from @thim81 - https://github.com/apideck-libraries/postman-to-k6/pull/91

But I noticed that there are zero packages published - https://github.com/orgs/apideck-libraries/packages?repo_name=postman-to-k6

So I went and looked at the Actions (CI) and saw https://github.com/apideck-libraries/postman-to-k6/actions/runs/4133989496

```
buildx failed with: ERROR: failed to solve: executor failed running [/bin/sh -c npm install --production]: exit code: 127
```

Thanks for considering! Have a wonderful day!

**Comment by mccurdyc** [comment]
https://github.com/apideck-libraries/postman-to-k6/pull/114

**Comment by thim81** [comment]
hi @mccurdyc 

Thanks for the PR, I approved it and is merged.
It will be part of the next release.

**Comment by thim81** [comment]
hi @mccurdyc 

We just released version 1.8.9, which includes your Docker PR.
Can you let me know, if the Docker is behaving as expected, and close the issue if it is OK.
